c++/src/kj/one-of.h

```diff
@@ -670,6 +671,27 @@ bool operator==(const OneOf<Variants...>& a, const OneOf<Variants...>& b) {
   return (_::compareIfIs<Variants>(a, b) || ...);
 }
 
+// TODO(someday) an ideal implementation would use kj::toCharSequence instead of kj::str,
+// producing a OneOf all the possible result types, and then would implement kj::_::fill()
+// for such a OneOf. This would avoid an extra copy and allocation when the OneOf is embedded
+// in a larger string.
+template <typename... Ts>
+requires (kj::Stringifiable<Ts> && ...)
+kj::String KJ_STRINGIFY(const kj::OneOf<Ts...>& o) {
+  kj::String result;
+  bool handled = false;
+
+  (( o.template is<Ts>() &&
+      ( result = kj::str(o.template get<Ts>()),
+        handled = true )
+    ), ...);
+
+  if (handled == false) {
+      return kj::str("Uninitialized OneOf");
```

Comment:
Too much indentation.

Also I'm going to bikeshed the message sorry.

```suggestion
    return kj::str("(null OneOf)");
```